### DISCUSSION TITLE
can gromacs be uese with deepmd-kit which is build on tensorflow 1.15

### DISCUSSION THREAD

**Discussion by ipe-zhangyz** [deepmodeling/deepmd-kit#2125] [Q&A]
I notice that in the doc of _Compile GROMACS with deepmd-kit_ it writes that "The C++ interface of deepmd-kit 2.x and tensorflow 2.x are required". As the .pb model file generated by tf1.15 can be used by tf2.x，I wonder if it is possible to use gromacs with deepmd-kit that is built on tf1.15?
Thanks!

**Comment by njzjz** [comment]
This might be a typo in the documentation.
